### ISSUE TITLE
Git installation doesn't work on OpenBSD

### ISSUE THREAD

**Issue by lucdig** [meganz/MEGAcmd#833]
I tried to compile the sorces downloaded with git on OpenBSD 7.3, but is fails:

# ./configure
gives this lines:

configure: Using SQLite3 as the default DB access layer.
checking sqlite3.h usability... no
checking sqlite3.h presence... no
checking for sqlite3.h... no
configure: error: sqlite3.h header not found or not usable

anyway, # find / -name sqlite3.h
gives me:

/usr/lib/sqlite3.h
/usr/local/include/sqlite3.h
/usr/local/libdata/perl5/site_perl/amd64-openbsd/auto/share/dist/DBD-SQLite/sqlite3.h

Is this a bug?

**Comment by brad0** [comment]
No, it's not a bug. You're not using the build infrastructure properly. Anyway, even when using it properly there are still some build issues I have an open PR for. There will be a port in the tree probably any day now.
